web/auth/user_from_auth.ex

```diff
@@ -105,12 +105,7 @@ defmodule PhoenixGuardian.UserFromAuth do
   defp auth_and_validate(auth, repo) do
     case repo.get_by(Authorization, uid: uid_from_auth(auth), provider: to_string(auth.provider)) do
       nil -> {:error, :not_found}
-      authorization ->
-        if authorization.token == auth.credentials.token do
-          authorization
-        else
-          {:error, :token_mismatch}
-        end
+      authorization -> authorization
```

Comment:
if authorization is found using UID and provider we simply return it without checking the tokens. Token checks prevents users from logging out and then logging back again, because tokens change between authorizations.